### PR TITLE
version1.1.2-2

### PR DESCRIPTION
**Pull request by TolisChal**
This PR:

- Removes loadSdpaFormatFile function.
- Changes the name of readSdpaFile() function to read_sdpa_file().
- Updates to CRAN version 1.1.2-2

### CODE REVIEW

File: cran_gen/NEWS.md [hunk 1]
```diff
@@ -29,3 +29,19 @@
...
 
+# volesti 1.1.2
+
+- Improve functionality of R function.
+- Use lower case and "_" separated names for the volesti's functions.
+- Use R classes for the convex polytopes
+
+# volesti 1.1.3
+
+- Remove loadSdpaFormatFile() function.
```
Comment (vissarion) [inline cran_gen/NEWS.md]: also readSdpaFormatFile

File: cran_gen/NEWS.md [hunk 2]
```diff
@@ -29,3 +29,19 @@
 
 * Fix CRAN warnings about deprecated use of ftime
 
+# volesti 1.1.2
+
+- Improve functionality of R function.
+- Use lower case and "_" separated names for the volesti's functions.
+- Use R classes for the convex polytopes
+
+# volesti 1.1.3
```
Comment (vissarion) [inline cran_gen/NEWS.md]: 1.1.2-2

File: cran_gen/NEWS.md [hunk 3]
```diff
@@ -29,3 +29,19 @@
...
+
+- Remove loadSdpaFormatFile() function.
+
+
```
Comment (vissarion) [inline cran_gen/NEWS.md]: why that many blank lines ?

**Review by vissarion — COMMENTED** [review]
Thanks for this PR!

To my understanding we only need 2 functions `load_sdpa_format_file` and `write_sdpa_format_file`. So we can drop `read_sdpa_format_file` right?

File: cran_gen/NEWS.md (hunk 1 above)
Comment (TolisChal) [reply cran_gen/NEWS.md]: I modified this. Thank you!

File: cran_gen/NEWS.md (hunk 2 above)
Comment (TolisChal) [reply cran_gen/NEWS.md]: Done.

File: cran_gen/NEWS.md (hunk 3 above)
Comment (TolisChal) [reply cran_gen/NEWS.md]: I removed them. Thanks.

**Review by vissarion — APPROVED** [review]

**Pull request merged**